connectors/sources/outlook.py

```diff
@@ -735,6 +823,13 @@ def get_default_configuration(cls):
                 "sensitive": True,
                 "type": "str",
             },
+            "client_emails": {
+                "depends_on": [{"field": "data_source", "value": OUTLOOK_CLOUD}],
+                "label": "Client Email Addresses (comma-separated)",
+                "order": 5,
```

Comment:
Because `exchange_server` is currently `"order": 5`, you'll need to bump that to 6, and `active_director_server` to 7, etc.